node/tests/swamp/config.go

```diff
@@ -0,0 +1,28 @@
+package swamp
+
+import (
+	"time"
+
+	"github.com/tendermint/tendermint/abci/types"
+	tn "github.com/tendermint/tendermint/config"
+
+	"github.com/celestiaorg/celestia-node/core"
+)
+
+// Config struct represents a set of pre-requisite attributes from the test scenario
+type Config struct {
+	App     types.Application
```

Comment:
i find it odd that the `app` is in the `Config` struct but ok. Maybe swamp should have a `InfraComponents` struct or something like that. I think `Config` is just a misnomer.

Reply:
Oh, thank you so much for the suggestion 🤩  . I'll change this name to InfraComponents in the upcoming PR